src/app/app.css

```diff
@@ -77,3 +77,117 @@ html, body, #root {
+  padding-left: 1.3em;
+  padding-right: 1.3em;
+}
+
+.automated-analysis-score-filter-slider .pf-c-slider__main {
+  margin-left: 1em;
+  --pf-c-slider__step--m-active__slider-tick--BackgroundColor: var(--pf-global--BorderColor--200);
+  --pf-c-slider__rail-track--before--fill--BackgroundColor: var(--pf-global--BorderColor--100);
+}
+
+.automated-analysis-score-filter-slider-ok .pf-c-slider__main {
+  --pf-c-slider__rail-track--before--base--BackgroundColor: var(--pf-global--success-color--100);
+  --pf-c-slider__step-tick--BackgroundColor: var(--pf-global--success-color--200);
+}
+
+.automated-analysis-score-filter-slider-warning .pf-c-slider__main {
+  --pf-c-slider__rail-track--before--base--BackgroundColor: var(--pf-global--warning-color--100);
+  --pf-c-slider__step-tick--BackgroundColor: var(--pf-global--warning-color--200);
+}
+
+.automated-analysis-score-filter-slider-critical .pf-c-slider__main {
+  --pf-c-slider__rail-track--before--base--BackgroundColor: var(--pf-global--danger-color--100);
+  --pf-c-slider__step-tick--BackgroundColor: var(--pf-global--danger-color--300);
+}
+
+.automated-analysis-score-filter-slider .pf-c-slider__value .pf-c-form-control {
+  width: 5em;
+}
+
+.clickable-automated-analysis-label-popover-header {
+  text-align: center;
+  margin-left: 16.6%;
+}
+
+.clickable-automated-analysis-label-popover-body {
+  text-align: center;
+}
+
+.clickable-automated-analysis-label-popover-body-score {
+  font-weight: bold;
+  margin-bottom: 0.5rem; 
+}
+
+.clickable-automated-analysis-label-popover-body-score.pf-m-danger {
+  color: var(--pf-c-popover--m-danger__title-text--Color);
+}
+
+.clickable-automated-analysis-label-popover-body-score.pf-m-warning {
+  color: var(--pf-c-popover--m-warning__title-text--Color);
+}
+
+.clickable-automated-analysis-label-popover-body-score.pf-m-success {
+  color: var(--pf-c-popover--m-success__title-text--Color);
+}
+
+.clickable-automated-analysis-label-popover-body-score.pf-m-default {
+  color: var(--pf-global--Color--100);
+}
+
+.clickable-automated-analysis-label-popover-body-score.pf-m-info {
+  color: var(--pf-c-popover--m-info__title-text--Color);
+}
+
+.clickable-automated-analysis-label-name {
+  max-width: 100%;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.pf-c-label-group.pf-m-vertical.pf-c-label-group {
+  height: 100%;
+  width: 100%;
+  background-color: var(--pf-global--Color--light-200);
+  border-color: var(--pf-global--BorderColor--100);
+  border-style: solid;
+  border-width: 0.1em;
+  border-radius: 0.5em;
+}
+
+.automated-analysis-grid-item .pf-c-label-group__main {
+  width: 100%;
+}
+
+.automated-analysis-grid-item .pf-c-label-group__list {
+  width: 100%;
+}
+
+.automated-analysis-grid-item .pf-c-label-group__list-item {
+  width: 100%;
+}
+
+.automated-analysis-grid-item .pf-c-label.pf-m-compact {
+  max-width: 100%;
+}
+
+.automated-analysis-grid-item .pf-c-label-group__label {
```

Comment:
We should also truncate LabelGroup title too as right now, there are overflows.

![Screenshot from 2022-12-01 14-20-14](https://user-images.githubusercontent.com/68053619/205142713-4594c91c-d545-4369-9bc9-d61d0b5b4103.png)


How about?

```css
.automated-analysis-grid-item .pf-c-label-group__label {
  font-weight: bold;
  max-width: 100%;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
}
```

Reply:
From another look, seems like the slider markers are overlapping each other. Is there any nice way around this? Or can we make the input field smaller when screen-size is small?